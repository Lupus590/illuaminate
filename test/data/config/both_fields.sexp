(pet
  (name "Dora")
  (age 23))
