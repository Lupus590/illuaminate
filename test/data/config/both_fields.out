{ Config.name = "Dora"; age = 23 }